### PR TITLE
fix: tables formatting with CJK characters

### PR DESCRIPTION
**Pull request by joeky888**
Table formatting is painful for CJK character users :(

Treat a CJK character with 2 spaces.

You'll need to install CJK fonts to get it work on your machine

img

### CODE REVIEW

File: gserver/test/data/features/after.format.feature [hunk 1]
```diff
@@ -7,10 +7,10 @@ Feature: Formatting feature
...
 
 		Given the following users exist:
-			| name   | email              | twitter         |
-			| Aslak  | aslak@cucumber.io  | @aslak_hellesoy |
-			| Julien | julien@cucumber.io | @jbpros         |
-			| Matt   | matt@cucumber.io   | @mattwynne      |
+			| name   | email              | twitter         | CJK Message 訊息    |
+			| Aslak  | aslak@cucumber.io  | @aslak_hellesoy | Yoo Hello world     |
+			| Julien | julien@cucumber.io | @jbpros         | Yoo 你好 世界       |
+			| Matt   | matt@cucumber.io   | @mattwynne      | Yoo こんにちは 世界 |
```
Comment (alexkrechik) [inline gserver/test/data/features/after.format.feature]: Looks like, that formatting still be wrong - total string length still be different in the last column:
Screen Shot 2019-07-06 at 1 36 25 PM

**Comment by joeky888** [comment]
You have to use a decent font, for example [Iosevka](https://github.com/be5invis/Iosevka/releases).

Make sure 2 english letters length is equal to a CJK character on your machine.

**Comment by joeky888** [comment]
Both on my browser and vscode works perfectly

img

**Comment by alexkrechik** [comment]
Yeah, but, ideally, it should works on all the PC's. Could we for ex. get CJK character for current font ?

**Comment by alexkrechik** [comment]
Font changing shouldn't destroy formatting for user.

**Comment by joeky888** [comment]
No, you need to install a correct font. 

This is an issue already discussed in vscode markdown addon https://github.com/yzhang-gh/vscode-markdown/issues/293#issuecomment-427635840

I am not a Chinese but they are talking about to use a correct font.

**Comment by tungtrinh** [comment]
This PR causes bug for all other unicode characters that been reported an issue here https://github.com/alexkrechik/VSCucumberAutoComplete/issues/328

Should be reverted.

**Pull request merged**